orientation/exercises/01_DICTIONARIES.md

```diff
@@ -37,4 +37,15 @@ purchases = [ ( 'GE', 100, '10-sep-2001', 48 ),
 
 Create a purchase history report that computes the full purchase price (shares times dollars) for each block of stock and uses the `stockDict` to look up the full company name. This is the basic relational database join algorithm between two tables.
 
+Example output for one block:
+`I purchased General Electric stock for $4800`
+
 Create a second purchase summary that which accumulates total investment by ticker symbol. In the above sample data, there are two blocks of GE. These can easily be combined by creating a dict where the key is the ticker and the value is the list of blocks purchased. The program makes one pass through the data to create the dict. A pass through the dict can then create a report showing each ticker symbol and all blocks of stock.
+
+Example output:
+```py
```

Comment:
Changed markdown parser from `py` to `sh`

Reply:
👍